source/api/annex/notes/design_patterns.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "IIIF Design Patterns"
+layout: spec
+tags: [annex, presentation-api, image-api]
+cssversion: 2
+---
+
+## Status of this Document
+{:.no_toc}
+
+This document is not subject to [IIIF's versioning semantics][iiif-semver]. Changes will be tracked within the document and its [internal change log][change-log]
+
+**Authors:**
+
+  * **[Michael Appleby](https://orcid.org/0000-0002-1266-298X)** [![ORCID iD](/img/orcid_16x16.png)](https://orcid.org/0000-0002-1266-298X), [_Yale University_](http://www.yale.edu/)
+  * **[Tom Crane](https://orcid.org/0000-0003-1881-243X)** [![ORCID iD](/img/orcid_16x16.png)](https://orcid.org/0000-0003-1881-243X), [_Digirati_](http://digirati.com/)
+  * **[Robert Sanderson](https://orcid.org/0000-0003-4441-6852)** [![ORCID iD](/img/orcid_16x16.png)](https://orcid.org/0000-0003-4441-6852), [_J Paul Getty Trust_](http://www.getty.edu/)
+  * **[Jon Stroop](https://orcid.org/0000-0002-0367-1243)** [![ORCID iD](/img/orcid_16x16.png)](https://orcid.org/0000-0002-0367-1243), [_Princeton University Library_](https://library.princeton.edu/)
+  * **[Simeon Warner](https://orcid.org/0000-0002-7970-7855)** [![ORCID iD](/img/orcid_16x16.png)](https://orcid.org/0000-0002-7970-7855), [_Cornell University_](https://www.cornell.edu/)
+  {: .names}
+
+{% include copyright2017.md %}
+
+----
+
+## Contents
+{:.no_toc}
+* Table of Discontent (will be replaced by macro)
+{:toc}
+
+## 1. Overview
+
+This document aims to describe, rationalize, and document the design principles that have evolved around the creation of the IIIF specifications to date. These principles should be used as a guide for ongoing and future work in order to promote consistency across the growing number of IIIF specifications.
+
+The principles do not speak to the process by which the specifications are written and managed over time.  For notes on the adopted approaches for these, please see:
+
+  * [Editorial Process][process]: The community process of discussion, and the editorial process of writing, the specifications.
+  * [Semantic Versioning][iiif-semver]: The interpretation of the semantic versioning pattern for managing change of APIs in a responsible and responsive manner.
+
+
+## 2. Design Principles
+
+### 2.1. Scope Design Through Shared Use Cases
+
+> [Discuss](https://github.com/IIIF/iiif.io/issues/1050)
```

Comment:
Yes, as a nod to "this is an ongoing discussion not the editors enforcing a set of laws".